### DISCUSSION TITLE
Is it possible to disable the wallpaper engine? (Resolved)

### DISCUSSION THREAD

**Discussion by endercreeper1000** [lxqt/lxqt#2350] [Q&A] [answered]
I am trying to find out a way to disable the current system LxQt uses for it's wallpaper and instead simply use xwallpaper instead (well really a script but that part isn't important). I can't find out if this is possible anywhere.

**Comment by ghost** [comment, marked as answer]
There is no 'wallpaper engine' but instead `pcmanfm-qt --desktop` draws a root window. To achieve using xwallpaper, it's necessary to disable Desktop under LXQt Modules.

**Comment by endercreeper1000** [comment]
thank you, that is EXACTLY what I have been looking for, I will now find a way to disable Modules

**Reply by ghost** [reply]
I lapsed and forgot to mention it is under LXQt Session. Cheers

**Reply by endercreeper1000** [reply]
Found it thank you once again this is exactly what I was looking for

**Comment by stefonarch** [comment]
I moved this to Q&A
